lessons/AI/ai-statistical-lang-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,452 @@
+@A{"She swallowed a fly"}
+
+@Q{Did everyone in your class end up with same phrase? How and why did that happen?}
+@A{Yes. When considering which word to generate next, there was always one word that was clearly the most probable, an no ties.}
+
+}
+
+@lesson-instruction{
+- Complete the second section of @printable-exercise{sampling.adoc}.
+}
+
+@QandA{
+@Q{What four-word phrase did you generate?}
+@A{The class should be split between "The spider to catch" and "The spider that wriggled".}
+
+
+@Q{Did everyone in your class end up with same phrase? How and why did that happen?}
+
+@A{No. Students arrived at different four-word phrases because we were forced to incorporate randomness when there was a tie for the most probable word to follow "spider".}
+}
+
+Modern statistical language models often invite users to adjust the @vocab{temperature} of the generated text, which influences the level of randomness. For instance, ChatGPT users are encouraged to use a _low_ temperature for more focused and less creative tasks. They are encouraged to use a _higher_ temperature for more random and increasingly creative tasks.
+
+@lesson-point{
+Temperature is the parameter that controls the randomness of the model's output as it generates text.
+}
+
+Even _without_ the ability to raise the temperature, we encountered randomness and variability in our generated texts. With a large enough corpus and a high enough temperature, a statistical language model will produce a new and unique output every single time!
+
+
+@strategy{AI "Hallucinations"}{
+
+As generative AI produces text, it often generates incorrect or misleading information. This is commonly known as an AI "hallucination".
+
+Some experts dislike this term and are encouraging an end to its use. These experts argue that _all_ output is "hallucinatory". Some of it happens to match reality... and some does not.
+
+The very same process that generates "hallucinatory" text _also_ generates the "non-hallucinatory" text. This truth helps us to understand _why_ it is so difficult to *fix* the "hallucination" problem.
+
+This term also attributes intent and consciousness to the AI, giving it human qualities when it is merely executing a program exactly as it is intended to do.
+}
+
+
+
+=== Synthesize
+
+@QandA{
+
+Critics of ChatGPT and other language models raise a variety of concerns. Consider each of them, below.
+
+@Q{This form of AI often has trouble with _facts_. Put another way, ChatGPT sometimes "makes stuff up." Why does this happen? What is actually going on?}
+
+@A{When ChatGPT produces false or misleading information, it is not glitching nor is there a bug. ChatGPT is just doing what it does, following the model as it ought to.}
+
+
+@Q{Others complain that ChatGPT has biases that can be seen in its text output. Where do these biases come from?}
+
+@A{If there are biases in the corpus, there will likely be biases in the output.}
+}
+
+
+
+
+
+== What is the Role of Language?
```

Comment:
This feels like a strange title for this section. I'm not sure that we answer that question through the work we're doing.